packages/errors/src/context.ts

```diff
@@ -575,6 +576,10 @@ export type SolanaErrorContext = DefaultUnspecifiedErrorContextToUndefined<
         [SOLANA_ERROR__TRANSACTION__ADDRESS_MISSING]: {
             index: number;
         };
+        [SOLANA_ERROR__TRANSACTION__EXCEEDS_SIZE_LIMIT]: {
+            transactionSize: number;
+            transactionSizeLimit: number;
```

Comment:
Nit: Maybe we could put `Bytes` on these field names?

Reply:
Good shout! How do you feel about a `number` type alias called `Bytes` on this file so TypeScript can provide that extra information? Or do you still think a more verbose name would be better for non-TypeScript users?